clusterloader2/testing/node-throughput/rc.yaml

```diff
@@ -24,6 +28,12 @@ spec:
       # Add not-ready/unreachable tolerations for 15 minutes so that node
       # failure doesn't trigger pod deletion.
       tolerations:
+      {{if .WindowsNodeTest}}
+      - key: "node.kubernetes.io/os"
+        operator: "Equal"
+        value: "win1809"
```

Comment:
1809 will be going away soon, we should switch to 2019 instead. Could be in this PR or a follow-up.

Reply:
I see, seems need to change to more relax operator? like  "Exists"